the-blue-alliance-ios/View Controllers/MyTBA/MyTBATableViewController.swift

```diff
@@ -67,6 +67,11 @@ class MyTBATableViewController<T: MyTBAEntity & MyTBAManaged, J: MyTBAModel>: TB
             NSSortDescriptor(key: #keyPath(MyTBAEntity.modelKey), ascending: true)
         ]
 
+        // Only show supported myTBA entities (basically, exclude team@event)
+        fetchRequest.predicate = NSPredicate(format: "%K IN %@",
```

Comment:
We should probably add some test around this...

Reply:
Note to Zach - we're adding a test to make sure a Team@Event doesn't show up in snapshot tests. All of the FRC stuff is private, which it feels wrong making it not-private, so we'll test it this way